docs/components/morphotagger.rst

```diff
@@ -82,8 +82,80 @@ You may also pass the tokenized sentences instead of raw ones:
     for parse in model(sentences):
         print(parse)
 
+If your data is large, you can call
+:func:`~deeppavlov.models.morpho_tagger.common.call_model` function, which will additionally
```

Comment:
I don't think this function should be there. We can just advise users to divide larger datasets into batches for inference:
```python3
from deeppavlov import build_model, configs
model = build_model(configs.morpho_tagger.UD2_0.morpho_ru_syntagrus_pymorphy, download=True)
sentences = ["Я шёл домой по незнакомой улице.", "Девушка пела в церковном хоре о всех уставших в чужом краю."]
batch_size = 32
for i in range(0, len(sentences), batch_size):
  batch = sentences[i:i+batch_size]
  for parsed_item in model(batch):
    print(parsed_item)
```

Reply:
I do not think it is the work of the user to create batches. This function was one of the most frequently used in my educational work with DeepPavlov, so I find it very convenient.